File: examples/openai-agents-sdk/chatbot/agent.py
"""
Minimal chatbot built with OpenAI Agents SDK deployed on Superserve.
"""

from agents import Agent, Runner

agent = Agent(
    name="assistant",
    instructions="You are a helpful assistant.",
)

while True:
    try:
        user_input = input()
    except EOFError:
        break
    result = Runner.run_sync(agent, user_input)
    print(result.final_output)

File: examples/typescript/openai-agents-sdk/chatbot/agent.ts
/**
 * Minimal chatbot built with OpenAI Agents SDK deployed on Superserve.
 */

import { Agent, run } from "@openai/agents"
import { createInterface } from "readline"

const agent = new Agent({
  name: "assistant",
  instructions: "You are a helpful assistant.",
})

const rl = createInterface({ input: process.stdin })
rl.on("line", async (line) => {
  const result = await run(agent, line)
  console.log(result.finalOutput)
})
